src/main/resources/db/migration/V1_20__add_sequence_to_description_addendum.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE if exists description_addendum;
+
+CREATE TABLE description_addendum
+(
+  id         SERIAL
+    CONSTRAINT description_addendum_pk PRIMARY KEY,
+  report_id  UUID         NOT NULL
+    CONSTRAINT description_addendum_report_fk REFERENCES report (id) ON DELETE CASCADE,
+  sequence   INTEGER      NOT NULL,
```

Comment:
other tables with `sequence` have default 0 – no biggy, just wondered if we care?

```sql
sequence               integer default 0 not null
```